client/src/deploy/send.rs

```diff
@@ -0,0 +1,40 @@
+use clap::{App, ArgMatches, SubCommand};
+
+use super::creation_common;
+use crate::{command::ClientCommand, common, RpcClient};
+
+pub struct SendDeploy;
+
+impl RpcClient for SendDeploy {
+    const RPC_METHOD: &'static str = "account_put_deploy";
+}
+
+enum DisplayOrder {
+    NodeAddress,
+    Input,
+}
```

Comment:
If we make the change suggested above, then `creation_common::input::arg()` won't take a display order.  For `common::node_address::arg()`, we could just use `creation_common::DisplayOrder::NodeAddress` to ensure consistency.  Which means this enum can be deleted.  Sorry for steering you in the wrong direction here!